modules/admin-ui/src/test/resources/test/unit/modules/events/subresources/controllers/toolsControllerSpec.js

```diff
@@ -1,5 +1,5 @@
 describe('Tools Edit controller', function () {
-    var $scope, $controller, $parentScope, $location, $window, $httpBackend, Notifications;
+    var $scope, $controller, $parentScope, $location, $httpBackend, Notifications;
```

Comment:
The changes to this file don't seem relevant to this PR at all.

Reply:
They are, though. The tests failed in an indescribably unhelpful manner because this test overrides the `$window` service with a very crippled version. It took me hours to find this. Trust me, this is very related. :expressionless: 

Also again: I think the "boy scout rule" should apply to this as well. :innocent:

Reply:
Ok, in that case the change is related to the PR then ;-)